test/hipcub/test_hipcub_device_radix_sort.hpp

```diff
@@ -1215,7 +1215,7 @@ inline void sort_keys_large_sizes()
 
     hipStream_t stream = 0;
 
-    const std::vector<size_t> sizes = test_utils::get_large_sizes(seeds[0]);
+    const std::vector<size_t> sizes = test_utils::get_large_sizes<34>(seeds[0]);
```

Comment:
Would be nice to document workarounds in the source as well and not impact Linux builds.

```suggestion
    // Workaround: `hipMalloc` always returns `hipSuccess` even when allocation fails.
    // We limit the maximum size so this bug doesn't occur.
#ifdef _WIN32
    const std::vector<size_t> sizes = test_utils::get_large_sizes<34>(seeds[0]);
#else
    const std::vector<size_t> sizes = test_utils::get_large_sizes(seeds[0]);
#endif
```